services/server/src/sourcify-chains-default.json

```diff
@@ -388,29 +388,6 @@
       }
     ]
   },
-  "44787": {
-    "sourcifyName": "Celo Alfajores Testnet",
-    "supported": true,
```

Comment:
Can you, for both chains, turn `supported: false`, just keep `sourcifyName` in addition, and remove other fields?